python/sglang/srt/layers/moe/topk.py

```diff
@@ -16,11 +16,10 @@
 
 import torch
 import torch.nn.functional as F
+from sgl_kernel import topk_softmax
```

Comment:
```python
if _is_cuda or _is_hip
```

Reply:
Mainly for compatibility with platforms other than NVIDIA and AMD.

Reply:
Make sense. I have updated it. @zhyncs